aviary/docs/user_guide/off_design_missions.ipynb

```diff
@@ -0,0 +1,92 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Off-Design Missions\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "Reserve missions are enabled for the following mission types:\n",
+    "\n",
+    "* height_energy    (completed)\n",
+    "* 2DOF (collocation) (complete)\n",
+    "* 2DOF (shooting)    (future work)\n",
+    "* height_energy (shooting)    (future work)\n",
```

Comment:
is this accurate?

Reply:
Yeah off-design works for height energy and 2DOF collocation. I assumed we will at some point add them into shooting as well but if that's not the case, I'll change it

Reply:
Sorry, I should have clarified. I think the 2dof shooting also works, and I'm not sure if the height energy shooting works. We should check that.

also, I just noticed that height energy says "completed" and 2dof says "complete", we should make those the same.